src/DotLiquid/StandardFilters.cs

```diff
@@ -643,11 +611,7 @@ public static string Date(Context context, object input, string format)
             }
             else if ((input is decimal) || (input is double) || (input is float) || (input is int) || (input is uint) || (input is long) || (input is ulong) || (input is short) || (input is ushort))
             {
-#if CORE
-                dateTimeOffset = DateTimeOffset.FromUnixTimeSeconds(Convert.ToInt64(input)).ToLocalTime();

```

Comment:
seems like this should have been adjusted to be used for .NET 6.0 and higher builds.

Reply:
Adjusted in what way? The subsequent line (which is left in) seems fine to me, and the tests pass on net461, coreapp2.0 and net6.0.